js/ad_manager_vast.js

```diff
@@ -2324,6 +2401,28 @@ OO.Ads.manager(function(_, $) {
 
     }, this);
 
+    var _handleWrapperAd = _.bind(function(ad, adLoaded) {
```

Comment:
can you clarify if it's amc ad or not? and can you use _getAdID function you created?